packages/x-data-grid-pro/src/hooks/features/rowReorder/useGridRowReorder.tsx

```diff
@@ -63,6 +64,16 @@ export const useGridRowReorder = (
   const ownerState = { classes: props.classes };
   const classes = useUtilityClasses(ownerState);
   const [dragRowId, setDragRowId] = React.useState<GridRowId>('');
+  const rows = gridExpandedSortedRowEntriesSelector(apiRef);
+  const lookup = React.useMemo(
```

Comment:
I'd prefer to off-load this to a selector as it'll remove the need for a method as well as ensure proper reactivity.
Wdyt?

Reply:
I created a new selector for it and wrapped it in `useGridSelector` while using it to ensure proper reactivity.